lib/index.js

```diff
@@ -25,7 +25,7 @@ var subtag = require('subtag');
  * a geographical point given as an object with latitude and longitude
  * properties. Search results closer to this point will be given
  * higher priority.
- * @param {Boolean} [options.trackProximity=true] If true, the geocoder proximity will automatically update based on the map view.
+ * @param {Boolean} [options.trackProximity=true] If true, the geocoder proximity will automatically update based on the map view for zoom levels 10 and above. To enable proximity bias for all zoom levels you need to set the `options.proximity` yourself.
```

Comment:
Sorry to be pedantic but technically this is any zoom > 9, which for example includes zoom `9.1`. I don't mind if we change the behaviour to be >= to make it clearer.

> "To enable proximity bias for all zoom levels you need to set the `options.proximity` yourself."
I'm hesitant to actually recommend this to users, since the Mapbox Geocoding API doesn't accept a zoom level in the proximity parameter when zoomed out at low zooms the proximity is more irrelevant.

What's the use case for passing a proximity even at low zooms?

Reply:
Ah yes true I guess it's "above 9"
Use case is we have a map that shows the 10 closest locations to the user and zoom out enough to show all 10 on screen. This might be a zoom level below 9.
We use IP based location so at times it can be inaccurate and pickup the ISP instead. As a user if I start typing my suburb I get results from other countries before results in the country that is on screen. I want locations closest to the center of the map still to take priority.
I feel like zoom level 9 is still very zoomed in and that something around 3+ would be more appropriate.

Reply:
The proximity used to only have an affect up to a few hundred meters, which is was the main driver behind only setting it >z9.

However I just checked carmen again now and it seems the logic has changed. https://github.com/mapbox/carmen/blob/0015a66dab6ba8a927cde06b744cb9991f15bd45/lib/util/proximity.js

Specifically addresses and pois are only affected by proximity weighted around z14, but country and region are weighted around z6.

So it looks like having a rough proximity even at low zooms is helpful for higher level results like localities, but I'm still concerned about an arbitrary map center having too much weight for low level results like addresses. Would be great if the Mapbox Geocoding Team could weigh in on this, or support zoom in the proximity bias.

Reply:
The map [here](https://docs.mapbox.com/search-playground/) works the way I would like.
If I first search "Australia" to center the map on the center of Australia.
I then search "Bris" I get "Brisbane, Queensland, Australia" or
If I search "107 Mullen" I get "107 Mullens Street, Balmain New South Wales 2041, Australia".
Both of these are exactly how I want it to work.
With mapbox-gl-geocoder I get results in other countries.

Reply:
Both of these are working as you prefer in the search playground precicly because the proximity isn't set. When you try at https://docs.mapbox.com/mapbox-gl-js/example/mapbox-gl-geocoder/ it's setting the proximity in Canada and so both of these searches aren't giving the same top results here.

ie. if you set the proximity at all zooms you'd be getting worse results when looking at a zoomed out map of the whole globe.

Reply:
If I customise the map to center on Australia like [this](https://codepen.io/codezeno/pen/LYYwoge?&editable=true) regardless of whether trackProximity is enabled or disabled or the zoom level. When I search "Bris" I get "Bristol, England, United Kingdom" as the first result. What is causing this difference compared to the playground map?

Reply:
it's the "language=en-US" which is causing the difference, mapbox-gl-geocoder sets it, the search playground doesn't (unless you turn it on). So it's not related to proximity.

Reply:
Ah you are absolutely right! This seems like flawed logic in the case of English.
How do I unset this? I have tried adding a IETF subtag but that didn't help and I cant find a way to unset it.

Reply:
Hmm seems you can't unset it, by default it uses "navigator.language || navigator.userLanguage || navigator.browserLanguage" https://github.com/mapbox/mapbox-gl-geocoder/blob/master/lib/index.js#L709

Let's cut this out to a new issue.